### DISCUSSION TITLE
Is there any way to decode this code?

### DISCUSSION THREAD

**Discussion by dongtataydoc** [mitmproxy/mitmproxy#6558] [Intercept Everything!]
#### Problem Description
i tried on burpsuite it works but mitmproxy doesn't.What's the problem?
![image](https://github.com/mitmproxy/mitmproxy/assets/61568682/dad84c3f-3522-4bd2-b647-a990a5fb22d1)

#### System Information
$ mitmproxy --version
Mitmproxy: 10.1.6
Python:    3.12.0
OpenSSL:   OpenSSL 3.1.4 24 Oct 2023
Platform:  Windows-10-10.0.19045-SP0

**Comment by Prinzhorn** [comment]
Please define "it works". I'm going to move this to discussions.
